src/base/_typography.scss

```diff
@@ -14,4 +14,65 @@ html {
 body {
   text-size-adjust: none; /* 1 */
   word-wrap: break-word; /* 2 */
+  font-family: 'Source Sans Pro', sans-serif;
+
+  @supports (font-variation-settings: normal) {
+    font-family: 'Source Sans Pro Variable', sans-serif;
+  }
+}
+
+@font-face {
```

Comment:
Can we include these `@font-face` declaration in a separate `_fonts.scss` file so they won't clutter our typographic styles?